Source/Core/DolphinQt2/Config/Graphics/GraphicsWidget.h

```diff
@@ -0,0 +1,39 @@
+// Copyright 2017 Dolphin Emulator Project
+// Licensed under GPLv2+
+// Refer to the license.txt file included.
+
+#pragma once
+
+#include <QHash>
+#include <QWidget>
+
+class QFormLayout;
+class QGroupBox;
+class QLabel;
+
+class GraphicsWidget : public QWidget
```

Comment:
Yeah sure, but (probably doesn't matter) why install an Event Filter on ``m_description_box`` did you mean ``m_enable_wireframe``?